LayoutTests/fast/parser/disable-frameset-ok-flag-inside-template.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<link rel=help href="https://html.spec.whatwg.org/multipage/syntax.html#parsing-main-inhead">
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<body>
+<script>
+var test = async_test("A frameset tag inside template should be simply ignored.");
+var iframe = document.createElement("iframe");
+iframe.onload = function() {
+  document.body.removeChild(iframe);
```

Comment:
It seems like this test is missing an assertion??

Reply:
I just took copy from 'source.chromium.org' and it seems that it does produce output without 'assertion' as well.

Reply:
But this test is missing the check that frameset is getting ignored. We need something like this:
```js
test.step(() => {
    assert_equals(iframe.contentDocument.querySelector('frameset'));
    assert_equals(iframe.contentDocument.querySelector('template').content.querySelector('frameset'));
})
```

Reply:
^ Let me test it locally. Doing multiple patch testing locally. :-)

Reply:
@rniwa - I just reverted this change and then tested locally and it fails without this patch and gives following error: `FAIL: Timed out waiting for notifyDone to be called`, so I think without 'assert_equals' in the test, it is still working as intended and shows that it needs to be fixed.

Reply:
ah, ok.